src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-roles.html

```diff
@@ -0,0 +1,137 @@
+<link rel="import" href="../../../../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../../../../bower_components/paper-input/paper-input.html">
+<link rel="import" href="../../../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="../../../../bower_components/paper-menu-button/paper-menu-button-animations.html">
+<link rel="import" href="../../../../bower_components/lancie-ajax/lancie-ajax.html">
+
+<link rel="import" href="../lancie-admin-page-layout.html">
+<link rel="import" href="../../lancie-admin-table/lancie-admin-table.html">
+
+
+<dom-module id="lancie-admin-roles">
+  <template>
+    <style include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        display: block;
+      }
+
+      paper-card {
+        width: calc(100% - 32px);
+        margin: 16px;
+      }
+
+      iron-icon {
+        margin-right: 10px;
+      }
+
+      [hidden] {
+        display: none !important;
+      }
+
+      @media (max-width: 640px) {
+        paper-card {
+          width: 100%;
+          margin: 0;
+        }
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="ajaxGetUsers" refurl="users" on-lancie-ajax="onResponse">
+    </lancie-ajax>
+    <lancie-ajax id="ajaxAddRole" method="POST" body="[[getBody(selectedRole)]]" refurl="users/[[selected.id]]/role"></lancie-ajax>
+    <lancie-ajax id="ajaxDeleteRole" method="POST" body="[[getBody(selectedRole)]]" refurl="users/[[selected.id]]/role/delete"></lancie-ajax>
+    
+    <paper-card heading="Roles">
+
+        <div class="card-content">
+          <p>Add roles to selected users</p>
+          <lancie-error id="userError"></lancie-error>
+          <lancie-admin-table id="table" column-mappings="[[mappings]]"></lancie-admin-table>
+          
+          <lancie-form id="addRoleForm" refurl="web/committee" on-response="onAddResponse">
+            <paper-dropdown-menu id="role" label="Roles">
+              <paper-listbox slot="dropdown-content" class="dropdown-content" on-iron-select="roleSelected">
+                <paper-item>Admin</paper-item>
+                <paper-item>Operator</paper-item>
+                <paper-item>Committee</paper-item>
+              </paper-listbox>
+            </paper-dropdown-menu>
+          </lancie-form>
+        </div>
+        <div class="card-actions">
+          <paper-button on-tap="addRoles">Add</paper-button>
+          <paper-button on-tap="deleteRoles">Delete</paper-button>
+        </div>
+    </paper-card>
+
+  </template>
+  <script>
+    (function () {
+      'use strict';
+
+      Polymer({
+        is: 'lancie-admin-roles',
+        properties: {
+          data: Object,
+          mappings: {
+            type: Object,
+            value: function() {
+              return {
+                profile: ['displayName'],
+              };
+            }
+          }
+        },
+
+        onResponse: function(e, request) {
```

Comment:
I would add resetting the error at this point, otherwise the error remains when the request succeeds. Usually we reset the error before the request is sent, so you could also do it at https://github.com/AreaFiftyLAN/lancie-admin/pull/75/files#diff-c8101382088cd073449421a040d06c7eR103